build.gradle

```diff
@@ -791,7 +791,6 @@ task testDocker {
 }
 
 task dockerUpload {
-  dependsOn distDocker
```

Comment:
Good question. The dependency was broken so that we could run multiple docker build jobs in parallel, or do anything we want between building and testing. It is purely for flexibility.

Reply:
What if you do a clean clone of the repo and just run `./gradlew testDocker` will it work?

Reply:
No, it will not. `test.sh` in the docker dir will fail. I actually think that is ok, and we have way too much docker stuff in the gradle build, which couples docker to gradle.  Trivial things like using a different container registry become non-trivial.